### ISSUE TITLE
Home - Language detection

### ISSUE THREAD

**Issue by robertocarroll** [robertocarroll/icc-beta#4]
See https://github.com/robertocarroll/icc-alpha/issues/14 for details

**Comment by robertocarroll** [comment]
No splash page. Detect language and remember preferences. 
![icc-beta-beyond-the-basics key](https://cloud.githubusercontent.com/assets/1565953/4309838/bbb20432-3ea0-11e4-9c66-8af681dcd675.jpg)

**Comment by robertocarroll** [comment]
Assigning this to SG to investigate SP 2013 Language detection features. Ability to redirect visitors to a site in an appropriate language based on the language setting of their web browser.

http://sharepointpromag.com/sharepoint-2013/top-10-sharepoint-2013-features-cheat-sheet

**Comment by simmigaba** [comment]
Yes its possible to re-direct visitors to a site in an appropriate language based on the language setting of their web browser. So in our case it will be one of 2 options - English or French as we will be creating a Multilingual website in these 2 languages. Please note that if we go for this option then any User who wish to switch from English to French or vice-versa can only do so by actually changing Language in the browser settings of whatever browser user is using and thereafter again loading the website in browser. This may sometime also require clearing of browser cookies or cache as there are instances when changing language setting in browser doesn't work. And there has to be some way to tell the user that they can change language by changing browser setting as any French User might still have browser language setting in English and may not know how to get French version of ICC website unless its communicated somehow from Homepage. 
Also if User browser has language setting other then French or English then we need to test that the website for default option (English) is opened. 

What do you think about language changing option which is more easy in existing website setup versus using browser setting which re-directs to appropriate language as per browser setting?
Thanks.
